src/pages/components/toggle-icon-button.mdx

```diff
@@ -130,10 +130,22 @@ import ToggleIconButtonTypesCode from '!!raw-loader!../../examples/button/Toggle
 
 ## API
 
+The ToggleIconButton component follows this structure:
+
+```jsx
+<Tooltip> {/* recommended */}
+  <ToggleIconButton />
```

Comment:
ditto re: trailing slash removal and `aria-label` recommendation

Reply:
Removed trailing slash! Added the `aria-label` recommendation to the respective descriptions instead of here - I've been trying to avoid adding any attributes to the structure to keep it as easily scannable as possible, but happy to revisit that if you think that would be valuable.

Reply:
Yeah, that totally makes sense. Thanks for driving for consistency.